src/reducers/authReducer.jsx

```diff
@@ -0,0 +1,63 @@
+const initState = {
+  authError: null,
+  errKey: null
+};
+
+const authReducer = (state = initState, action) => {
+  switch (action.type) {
+    case "LOGIN_ERROR":
+      return {
+        ...state,
+        authError: "Log in failed..!",
```

Comment:
the two dots after the error message here and in other places is a bit weird. i would just do it with the exclamation point `Log in failed!`